packages/strapi-design-system/src/Combobox/Combobox.js

```diff
@@ -250,7 +251,11 @@ export const Combobox = ({
         {value}
       </VisuallyHidden>
       <Stack spacing={label || hint || error ? 1 : 0}>
-        {label && <FieldLabel id={labelId}>{label}</FieldLabel>}
+        {label && (
+          <FieldLabel required={required} id={labelId}>
```

Comment:
Good catch! Could you also check if the `labelAction` is passed through as `action={labelAction}`? It seems to be missing too.

Reply:
Nice catch! Pushed 👍🏽